macosx/README.md

```diff
@@ -0,0 +1,23 @@
+# Building a Mac OS X Package
+
+
+To build the distribution package for Gregorio for Mac, you first need to install [Packages](http://s.sudre.free.fr/Software/Packages/about.html) and [Pandoc](johnmacfarlane.net/pandoc/index.html).  Once those are installed, simply run:
+
+`create_pacakge.sh`
+
+This script contains all the necessary commands to create the distribution package.  It will first build and "install" the package into the `macosx` directory (this way we know where everything is).  It will then use Pandoc to convert the necessary documentation files from markdown to ??? (the format required by Packages).  Once that's done, it will use the command line interface for Packages to build the installer package.
+
+The distribution package is composed of several raw packages, each responsible for installing different things:
+
+1. gregorio-base (embedded in the main installer): This installs the command line tool and the GregorioTeX files.  These are the required components and must be installed.
+2. gregorio-examples: This installs the examples folder.  This is an optional install (installed by default).
+3. gregorio-contrib: This installs the contrib folder.  This is an optional install (installed by default).
+4. gregorio-doc: This installs the documentation (doc folder).  This is an optional install (installed by default).
```

Comment:
I understood that, but I just don't really see the point... installing 130ko shouldn't be a problem for anyone... that's what is done by the Windows installer, the debian packages, etc. and I think it's quite ok...